[refman] Move chapters into new structure. (#11601)
coq/coq: doc/sphinx/libraries.rst

=====================
Libraries and plugins
=====================

Coq is distributed with a standard library and a set of internal
plugins (most of which provide tactics that have already been

> presented in Chapter :ref:`writing-proofs`).  The current chapter

Reviewer: ```suggestion
presented in Chapter :ref:`writing-proofs`).  This chapter
```

presents this standard library:

.. toctree::
   :maxdepth: 1

   language/coq-library
   addendum/miscellaneous-extensions

In addition, Coq has a rich ecosystem of external libraries and

> plugins.  Such libraries and plugins can be browsed online through the

Reviewer: ```suggestion
plugins.  These libraries and plugins can be browsed online through the
```

`Coq Package Index <https://coq.inria.fr/opam/www/>`_ and installed

> thanks to the `opam package manager

Reviewer: ```suggestion
with the `opam package manager
```
